### PR TITLE
Set up Helm+Docker repo and update readme files

### PR DESCRIPTION
**Pull request by tommyongib**
- Add make files to support setting up Helm repo within this Github repo (will do a release in next PR)
- Add instructions on how to release
- Add instructions on how to do one-time setup
- Add instructions on how to deploy and troubleshoot

Following similar approach of https://github.com/infobloxopen/cert-manager

### CODE REVIEW

File: Makefile.vars [hunk 1]
```diff
@@ -0,0 +1,4 @@
+RELEASE_VERSION ?=$(shell grep version colo/charts/farsight-collector-colo/Chart.yaml | awk '{print $$2}')
+
+IMAGE_NAME_COLO ?= farsight-collector-colo
+IMAGE_REGISTRY  ?= infoblox
```
Comment (hdoddikindi) [inline Makefile.vars:4]: Is this the public docker repo?

Comment (tommyongib) [reply Makefile.vars:4]: yes, it is public https://hub.docker.com/r/infoblox/farsight-collector-colo

**Review by hdoddikindi — APPROVED** [review]
LGTM

**Review by ssarkar-ib — APPROVED** [review]
LGTM

**Pull request merged**